pennylane/templates/subroutines/select.py

```diff
@@ -167,9 +166,9 @@ def ops(self):
         return self.hyperparameters["ops"]
 
     @property
-    def control_wires(self):
+    def control(self):
         """The control wires."""
-        return self.hyperparameters["control_wires"]
+        return self.hyperparameters["control"]
```

Comment:
Do we need this property?

Reply:
It's not mandatory. We can remove it. Users could do `op.hyperparameters['control']`
Is that the preference?

Reply:
I don't have a strong preference, but curious to see what @albi3ro thinks.